src/compiler/emitter.ts

```diff
@@ -6107,7 +6107,14 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
                         if (namespaceDeclaration && !isDefaultImport(node)) {
                             // import x = require("foo")
                             // import * as x from "foo"
-                            if (!isExportedImport) write("var ");
+                            if (!isExportedImport) {
+                                if (languageVersion <= ScriptTarget.ES5) {

```

Comment:
Can you wrap this in a helper method, since you repeat this pattern 3 times in this PR